gateway-js/src/index.ts

```diff
@@ -201,9 +202,9 @@ export class ApolloGateway implements GraphQLService {
   // how often service defs should be loaded/updated (in ms)
   private experimental_pollInterval?: number;
   // Configure the endpoint by which gateway will access its precomposed schema.
-  // For now, `null` is default and means to continue using the legacy managed mode.
+  // `null` will revert the gateway to legacy mode (polling GCS and composing the schema itself).
```

Comment:
probably be slightly more explicit: string means use that endpoint, null means use GCS, undefined means not managed federation.